test/test_api.py

```diff
@@ -53,10 +53,6 @@ def test_open_dataset(self):
                         constants.NNODES_outCSne30)
         nt.assert_equal(len(uxds_var2_ne30.uxgrid._ds.data_vars),
                         constants.DATAVARS_outCSne30)
-        nt.assert_equal(uxds_var2_ne30.uxgrid.source_grid,
-                        str(self.gridfile_ne30))
-        nt.assert_equal(uxds_var2_ne30.source_datasets,
```

Comment:
Why removing this?

Reply:
It was removed due to `source_datasets` not existing, but once it's added back I'll reintroduce this check.

Reply:
Actually `uxds_var2_ne30.source_datasets` exists.

Reply:
Reintroduced